src/fields/Field.cpp

```diff
@@ -1,13 +1,9 @@
 #include "Field.h"
 
-Field::Field(string fieldName, FieldType fieldType, bool isPrimaryKey, bool isUnique, bool isNullable, bool isAutoFilled) {
-    _fieldName = fieldName;
-    _fieldType = fieldType;
-    _isPrimaryKey = isPrimaryKey;
-    _isUnique = isUnique;
-    _isNullable = isNullable;
-    _isAutoFilled = isAutoFilled;
-}
+Field::Field(string &fieldName, FieldType fieldType, bool isPrimaryKey, bool isUnique, bool isNullable, bool isAutoFilled) :
+    _fieldName(fieldName), _fieldType(fieldType), _isPrimaryKey(isPrimaryKey), _isUnique(isUnique), _isNullable(isNullable), _isAutoFilled(isAutoFilled) {
+        _isNull = true;

```

Comment:
Ok, never mind. This had to do with the order these appeared in `Field.h`, so it can also be initialized in the list.
Done